### PR TITLE
Read config kafka

### PR DESCRIPTION
**Pull request by aliok**
Fixes #

## Proposed Changes

- Read config-kafka configmap and get the broker address from there
- Needed to make a func in eventing-kafka public and use it for parsing the configmap data: https://github.com/knative-sandbox/eventing-kafka/pull/847
- Upgraded eventing-kafka dep to main branch. This also affected the eventing dependency version too.

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

**Comment by aliok** [comment]
cc @devguyio @matzew

**Comment by aliok** [comment]
/test pull-knative-sandbox-eventing-kafka-broker-integration-tests

```
Failed to clean the resource "br" : timed out waiting for the condition
```

File: control-plane/pkg/reconciler/channel/channel.go [hunk 1]
```diff
@@ -203,8 +225,20 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, channel *messagingv1beta1
 	return nil
 }
 
+func (r *Reconciler) channelConfigMap() (*corev1.ConfigMap, error) {
```
Comment (matzew) [inline control-plane/pkg/reconciler/channel/channel.go:228]: we might change that name later ... once I started w/ the refactoring on all sorts of `reconcileXzy` (like distributed channel)

Comment (aliok) [reply control-plane/pkg/reconciler/channel/channel.go:228]: sure, sounds good

**Comment by pierDipi** [comment]
Hi Ali, I want to share what's happening here. 

Basically, the update to the eventing dependencies brings the new test Matthias wrote, see https://github.com/knative-sandbox/eventing-kafka-broker/pull/1194/files#diff-64634a925a6f4c9df07657597f4f3e08dca34517b71b2c42c7eb779406f05880R74.

However, the eventing webhook update is part of the nightly job update in https://github.com/knative-sandbox/eventing-kafka-broker/pull/1195. 

Now #1195 is in, so we should be able to have a green run. (Prow rebases the PR automatically)

/retest

**Comment by pierDipi** [comment]
Different issue https://github.com/knative/eventing/issues/5700

> test_runner.go:163: Couldn't initialize clients: unable to find available namespace

/retest

**Comment by aliok** [comment]
thanks @pierDipi !

**Pull request merged**